src/docs/product/cli/dif.mdx

```diff
@@ -9,6 +9,10 @@ information about crash reports for most compiled platforms. `sentry-cli` can be
 used to validate and upload debug information files. For more general
 information, refer to [_Debug Information Files_](/workflow/debug-files/).
 
+## Permissions
+
+`sentry-cli` requires an Auth Token to be authenticated with a set of [Permissions & Scopes](/api/permissions/) to be able to upload Debug Information Files and for that you must have either a `project:releases` or `project:write` Scope.
```

Comment:
Added the word "the" to the beginning of the sentence to avoid beginning the sentence with a lower case letter. The sentence was also a bit too long and had some repeated words, so I've tried to reword a bit. 

```suggestion
The `sentry-cli` requires an Auth Token to be authenticated with a set of [Permissions & Scopes](/api/permissions/) so that Debug Information Files can be uploaded. To do this, you must have either a `project:releases` or `project:write` Scope.
```

Reply:
great @imatwawana thank you!